test-utils/src/main/scala/com/mesosphere/utils/http/AkkaHttpResponse.scala

```diff
@@ -0,0 +1,37 @@
+package com.mesosphere.utils.http
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
+import akka.stream.Materializer
+import play.api.libs.json.Reads
+
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.{ExecutionContext, Future}
+import scala.reflect.ClassTag
+
+object AkkaHttpResponse {
```

Comment:
do we need this to test library? There might not even be akka http, right?

Reply:
oh I see, we use this to talk to mesos API, that was not really obvious at first sight to me :)